ezgui/src/canvas.rs

```diff
@@ -11,6 +11,8 @@ const DRAG_THRESHOLD: f64 = 5.0;
 
 const PAN_SPEED: f64 = 15.0;
```

Comment:
The auto-pan behavior interacts really unexpectedly with the main event loop. `handle_event` is pretty much called for every raw event (key presses, any mouse input, and update events when the simulation is running). So try auto-panning, then press space to pause the sim -- auto-panning will stop! And if you wiggle the mouse while auto-panning to generate more events, the panning speeds up. We need to make this behaviour independent of all these other events.

I'm... actually not sure of an easy way to deal with this. The main event loop waits for external input unless the simulation is running or something else in `game` returns `EventLoopMode::Animation` to trigger regular update events. We don't want to mix up update events for auto-pan and running the sim. It might be time to make this API a little smarter and explicitly schedule named events to happen in the future, something like `ctx.schedule_event(Duration::millis(100), "auto-pan")` and changing `nonblocking_is_update_event` to plumb this name and dt. I think we can solve this next after this PR; how about we just disable the new behavior by default, fix the event loop, then enable it?

Reply:
That sounds good, I've changed it to be disabled by default.